app/sitemap.ts

```diff
@@ -4,9 +4,7 @@ import { headers } from 'next/headers';
 
 import { app, environment, routing } from '@/adapters/server';
 
-const MINUTE = 60;
-
-export const revalidate = 15 * MINUTE;
+export const revalidate = 900;
```

Comment:
IMO previous code with additional constant was easier to understand 🙂

Reply:
I'll add a comment to explain this as Next 15 is strict on value been just a number, using an expression fails
<img width="672" alt="Screenshot 2025-01-23 at 7 27 58 AM" src="https://github.com/user-attachments/assets/d6afebda-f611-4742-8948-64537f993273" />

Reply:
https://github.com/prezly/theme-nextjs-bea/pull/1213/commits/865d0328440bfb1aba3dd00cd12a25c3ea06d86d